example-studio-all-fashion-mnist/generator.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+set -eu
+
+if [ "$#" -ne 5 ]; then
+    echo "\n\nERROR: 
+    You must provide the following arguments:
+    1. Git account or org name (eg, tapadipti)
+    2. Repo name (eg, fashion_mnist)
+    3. Git author name (eg, tapadipti)
+    4. Git author email (eg, tapadipti@gmail.com)
+    5. DVC remote URL (eg, s3://mydvc/tapa)
+    Eg, sh generator.sh tapadipti fashion_mnist tapadipti tapadipti@gmail.com s3://mydvc/tapa
+    The repo will be created at https://github.com/GIT_ACC_OR_ORG/REPO_NAME (eg, https://github.com/tapadipti/fashion_mnist).
+    If this repo already exists, it will be replaced by the new one - so make sure that you don't need the existing one.
+    Make sure you have the required access to the Git account and DVC remote location.\n\n"
+    exit
+fi
+
+GIT_ORG=$1
+REPO_NAME=$2
+GIT_AUTHOR_NAME=$3
```

Comment:
We are using "Olivaw" for the name and the email. You can see the other [`generate.sh`](https://github.com/iterative/example-repos-dev/blob/master/example-dvc-experiments/generate.bash#L21) for this.